l10n/en/products/vpn/platform-post-download.ftl

```diff
@@ -0,0 +1,65 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/mac-download/
+
+vpn-mac-download-page-title = Download { -brand-name-mozilla-vpn } for { -brand-name-mac-short }
+vpn-windows-download-page-title = Download { -brand-name-mozilla-vpn } for { -brand-name-windows }
+vpn-mac-downloading-mozilla-vpn = Downloading { -brand-name-mozilla-vpn } for { -brand-name-mac-short }
+vpn-windows-downloading-mozilla-vpn = Downloading { -brand-name-mozilla-vpn } for { -brand-name-windows }
+
+# Variables:
+#   $url (url) - link for either Windows or Mac download, depending on user platform
+#   $id (string) - unique ID for the anchor tag
+# The <br> tag is a line break for the HTML text
+vpn-your-download-should-start = Your { -brand-name-mozilla-vpn } download should start automatically. If it doesn’t, <br> <a href="{ $url }" id="{ $id }">restart the download</a>, or go back to download for another device.
+
+vpn-open-the-file = 1. Open the file
+vpn-mac-go-to-your-downloads = Go to your downloads folder and double click the .pkg file
+vpn-windows-go-to-your-downloads = Go to your downloads folder and double click the .msi file
+vpn-launch-the-app = 2. Launch the app
+vpn-mac-follow-the-steps = Follow the steps outlined in the { -brand-name-mozilla-vpn } installer to start the app
+vpn-windows-a-progress-bar = A progress bar will appear and the installation process will begin
+vpn-login-or-signup = 3. Log in or sign up
+vpn-enter-your-email = Enter your email address to log in or sign up and get started
+vpn-turn-on-vpn = 4. Turn on VPN
+vpn-pick-a-recommended = Pick a recommended location for a fast, stable connection or choose your desired location manually, then switch the toggle ON
+
+vpn-download-faq-need-help = Need help getting started?
+vpn-download-faq-here-are-some = Here are some of the most common questions about setting up { -brand-name-mozilla-vpn }
+vpn-download-faq-why-use = Why should I use { -brand-name-mozilla-vpn }
+
+# Variables:
+#   $faq (url) - link to https://www.mozilla.org/en-CA/products/vpn/more/what-is-a-vpn/
+vpn-download-faq-as-more-daily = As more of daily life takes place through the internet, online privacy and security become even more important. A { -brand-name-vpn }, <a href="{ $faq}">Virtual Private Network</a>, creates an encrypted tunnel between your devices and the internet, ensuring your connection remains secure and private while protecting you in two important ways:
```

Comment:
Fair point. I was hesitant to use the `brand-name-vpn` variable, but since I noticed that it existed, I assumed it could be used in this circumstance, but I'm happy to remove it.